packages/editor/src/components/post-title/index.js

```diff
@@ -180,10 +179,7 @@ function PostTitle( _, forwardedRef ) {
 				...create( { html: title } ),
 				...selection,
 			};
-			const newValue = insert(
-				value,
-				create( { html: stripHTML( content ) } )
-			);
+			const newValue = insert( value, create( { html: content } ) );
```

Comment:
Not sure if we should revert this. [Discussion here](https://github.com/WordPress/gutenberg/pull/35825#discussion_r1018329323)

Reply:
Actually if we are now rendering the HTML it's ok to remove `stripHTML` here as HTML _is_ valid in Post Titles.

However we _must_ ensure that the generated slug does _not_ contain HTML.

Reply:
<img width="1260" alt="Screen Shot 2023-09-29 at 10 36 31" src="https://github.com/WordPress/gutenberg/assets/444434/80f3ce97-7376-4939-a361-bca8c8e41288">

I tested and the slug does not contain HTML.

Reply:
Original PR to add this was in https://github.com/WordPress/gutenberg/pull/35825